students/student_framllat/session02/fizzbuzz.py

```diff
@@ -0,0 +1,15 @@
+import random
+
+for i in range(1,10):
+
+    test_num = random.randint(1,100)
+#    print("This is the num:  ",test_num)
+
+    if (test_num % 3 == 0) and (test_num % 5 == 0):
```

Comment:
this is a good, solid solution.

but now that you are ahead of the game, can you figure out how not to check for `% 3` and `% 5` twice each?